pkg/manifest/manifest.go

```diff
@@ -398,16 +398,14 @@ func ValidManifest(m Manifest) error {
 	if m.ID() == "" {
 		return fmt.Errorf("manifest must contain an 'id'")
 	}
-	for key, stanza := range m.GetLaunchableStanzas() {
+	for launchableID, stanza := range m.GetLaunchableStanzas() {
 		switch {
 		case stanza.LaunchableType == "":
-			return fmt.Errorf("'%s': launchable must contain a 'launchable_type'", key)
-		case stanza.LaunchableId == "":
```

Comment:
If we desire a less risky change, we could probably make this validation fail if the two notions of launchable ID differ and address any violations case-by-case